pyprojectsort/main.py

```diff
@@ -57,6 +58,11 @@ def reformat_pyproject(pyproject: dict | list) -> dict:
     return pyproject
 
 
+def _check_format_needed(original: dict, reformatted: dict) -> bool:
+    """Check if there are any differences between original and reformatted. """
```

Comment:
Good idea creating this function! I just noticed an extra 'space' character at the end of the docstring. Also, would it be possible to add a unit test for calling this function on its own?

```suggestion
    """Check if there are any differences between original and reformatted."""
```

Reply:
Added a test for the function also